seq_logger.js

```diff
@@ -51,6 +51,36 @@ class SeqLogger {
         return this._ship();
     }
 
+    // A browser only function that queues events for sending using the
+    // navigator.sendBeacon() API.  This may work in an unload or pagehide event
+    // handler when a normal flush() would not.
+    // Events over 63K in length are discarded (with a warning sent in its place) 
+    // and the total size batch will be no more than 63K in length.
+    flushToBeacon() {
+        if (this._queue.length === 0) {
+            return false;
+        }
+
+        if (typeof navigator === 'undefined' || !navigator.sendBeacon || typeof Blob === 'undefined') {
+            return false;
+        }
+
+        const currentBatchSizeLimit = this._batchSizeLimit;
+        const currentEventSizeLimit = this._eventSizeLimit;
+        this._batchSizeLimit = 63 * 1024;
+        this._eventSizeLimit = 63 * 1024;
```

Comment:
These should be the `min()` of the current batch/event size limits, since this might end up increasing the limits, and therefore lead to a rejected event on the Seq end.

Reply:
I've added the `Math.min()` code to these lines.